Eastern-Research-Group/mywaterway
# Feature/allow direct routing to plan summary page

## Related Issues:
* [https://app.breeze.pm/projects/100762/cards/3077190](https://app.breeze.pm/projects/100762/cards/3077190)

## Main Changes:
* Updated the community to plan-summary link to use the organizationId attribute. 
* Removed the intermediary plan-summary (OrgSelect) page. 
* Removed an error message that was only used on the deleted OrgSelect component.

## Steps To Test:
1. Navigate to [http://localhost:3000/community/san%20antonio%20tx/restore](http://localhost:3000/community/san%20antonio%20tx/restore)
2. Hover over the "Open Plan Summary" link. The url that shows up in the bottom left should include the organization id (i.e. [http://localhost:3000/plan-summary/TCEQMAIN/1](http://localhost:3000/plan-summary/TCEQMAIN/1))
3. Click the link and make sure goes directly to the plan-summary page.
4. Run the cypress test and make sure it works.
